static/src/js/components/GranuleResults/__tests__/GranuleResultsItem.test.js

```diff
@@ -335,316 +363,417 @@ function setup(type, overrideProps) {
+      const { title, granuleThumbnail } = granule
+
+      expect(EDSCImage).toHaveBeenCalledTimes(1)
+      expect(EDSCImage).toHaveBeenCalledWith(
+        expect.objectContaining(
+          {
+            alt: `Browse Image for ${title}`,
+            className: 'granule-results-item__thumb-image',
+            height: 85,
+            isBase64Image: true,
+            src: granuleThumbnail,
+            width: 85,
+            useSpinner: false
+          }
+        ),
+        {}
+      )
     })
 
     test('renders the start and end date', () => {
-      const { enzymeWrapper } = setup('opensearch')
+      setup('opensearch')
 
-      expect(enzymeWrapper.find('.granule-results-item__temporal--start').find('h5').text()).toEqual('Start')
-      expect(enzymeWrapper.find('.granule-results-item__temporal--start').find('p').text()).toEqual('2019-04-28 00:00:00')
-      expect(enzymeWrapper.find('.granule-results-item__temporal--end').find('h5').text()).toEqual('End')
-      expect(enzymeWrapper.find('.granule-results-item__temporal--end').find('p').text()).toEqual('2019-04-29 23:59:59')
+      expect(screen.getByRole('heading', { name: 'Start' })).toBeInTheDocument()
+      expect(screen.getByText('2019-04-28 00:00:00')).toBeInTheDocument()
+      expect(screen.getByRole('heading', { name: 'End' })).toBeInTheDocument()
+      expect(screen.getByText('2019-04-29 23:59:59')).toBeInTheDocument()
     })
 
-    test('disables the Add granule button', () => {
-      const { enzymeWrapper } = setup('opensearch')
+    test('Add granule button is disabled', () => {
+      setup('opensearch')
 
-      expect(enzymeWrapper.find('.granule-results-item__button--add').props().disabled).toBeTruthy()
+      expect(screen.getByRole('button', { name: 'Add granule' })).toBeDisabled()
     })
   })
 
-  describe('when clicking the remove button', () => {
+  describe('when clicking the Filter Granule button', () => {
     describe('with CMR granules', () => {
-      test('it removes the granule from results', () => {
-        const { enzymeWrapper, props } = setup('cmr')
-        const removeButton = enzymeWrapper.find(MoreActionsDropdownItem).at(1)
+      test('it removes the granule from results', async () => {
+        const { props, user } = setup('cmr')
+
+        await act(async () => {
+          const moreActions = screen.getByRole('button', { name: 'More actions' })
+          await user.click(moreActions)
+        })
+
+        await act(async () => {
+          await userEvent.click(screen.getByRole('button', { name: 'Filter granule' }), { pointerEventsCheck: 0 })
+        })
 
-        removeButton.simulate('click')
         expect(props.onExcludeGranule.mock.calls.length).toBe(1)
         expect(props.onExcludeGranule.mock.calls[0]).toEqual([{
           collectionId: 'collectionId',
           granuleId: 'granuleId'
         }])
-
-        expect(removeButton.props().title).toEqual('Filter granule')
```

Comment:
I think we do want to keep this assertion just moved to RTL

Reply:
Looking back, checking that the granule filters how the `1 Granule Filtered` etc is actually also being done in `static/src/js/components/GranuleFilters/__tests__/GranuleFiltersForm.test.js`